doc/source/quickstart.rst

```diff
@@ -246,6 +260,26 @@ output format and to provide the best looking image. For more information
 on saving datasets and customizing enhancements see the documentation on
 :doc:`writers`.
 
+
+Slicing and subsetting scenes
+=============================
+
+Array slicing can be done at the scene level in order to get excerpts with consistent navigation throughout. Note that this does not take into account scenes that may include multi-resolution channels.
+
+  >>> scene_slice = global_scene[2000:2004, 2000:2004]
+  >>> vis006_slice = scene_slice['VIS006']
+  >>> vis006_slice_meas = vis006_slice.values
+  >>> vis006_slice_lon, vis006_slice_lat = vis006_slice.attrs['area'].get_lonlats()
+
+To subset multi-resolution data, use the `.crop` function. 
```

Comment:
Cay you make this a link to the `crop` method ?

Reply:
You should be able to do:

```
:meth:`~satpy.scene.Scene.crop`
```

The `~` will make it only say "crop" in the actual text.

Reply:
Added.